src/v3/bandwidthRtc.ts

```diff
@@ -433,37 +433,67 @@ export class BandwidthRtc {
   }
 
   private setupNewPeerConnection(peerConnection: RTCPeerConnection, onPeerClosed: CallableFunction): void {
-    peerConnection.onconnectionstatechange = (event: Event) => {
-      const pc = event.target as RTCPeerConnection;
-      logger.debug("onconnectionstatechange", pc.connectionState, pc);
-      const connectionState = pc.connectionState;
-      if (connectionState === "disconnected") {
-        logger.warn("Peer disconnected, connection may be reestablished");
-      }
-      if (connectionState === "failed" || connectionState === "closed") {
-        logger.warn("Connection lost, refresh to retry");
-        // TODO: make automatic reconnection work
-        // onPeerClosed();
+    peerConnection.onconnectionstatechange = (event) => {
+      try {
+        const pc = event.target as RTCPeerConnection;
+        logger.debug("onconnectionstatechange", pc.connectionState, pc);
+        const connectionState = pc.connectionState;
+        if (connectionState === "disconnected") {
+          logger.warn("Peer disconnected, connection may be reestablished");
+        }
+        if (connectionState === "failed" || connectionState === "closed") {
+          logger.warn("Connection lost, refresh to retry");
+          // TODO: make automatic reconnection work
+          // onPeerClosed();
```

Comment:
can remove commented out code?

Reply:
Ah yeah I keep forgetting it doesn't populate the title with the commit message; I swear it either used do or does on some repos.

As to the commented out code, it will be fixed this week and I want to make sure not to forget that that's what needs to be called here.